docs/schemes.md

```diff
@@ -52,6 +52,162 @@ Description of the files:
 * `pk.pem` is the scheme's public key against which `index.sig` should verify.
 * `kss-0.pem` (optional): if the scheme uses a [keyshare server](keyshare-protocol.md), then this contains the keyshare server's public key.
 
+IRMA schemes have the following nested structure: a scheme contains (multiple) issuers, which contain (multiple) credentials, which contain (multiple) attributes. Each of these are identified and referred to using a string that we call the *identifier*. In the case of schemes, issuers and credentials, these can be seen in the directory structure. In the directory tree above:
+
+* The identifier of the scheme is `scheme-id`,
+* This scheme contains an issuer whose identifier is `issuer-id`,
+* This issuer contains a credential type whose identifier is `credentialtype-id`.
+
+The `description.xml` of a credential type contains the definitions, including their identifiers, of the attributes contained within that credential type.
+
+These identifiers are used to refer to these entities within [IRMA session requests](session-requests.md), joined together using a dot `.` as separator. For example, [below](#credential-types) is shown the `description.xml` of the credential type `credentialtype-id` included in the directory tree above. This file contains an attribute whose identifier is `boolean-attr`. Then the following would be a valid IRMA disclosure request that requests an IRMA app user to disclose this attribute:
```

Comment:
You mean to explicitly refer to the `id` XML attribute that defines that identifier? Then how about this:

``This file contains an attribute whose identifier, as defined by the `id` XML attribute, is `boolean-attr`.``

Reply:
i actually meant getting rid of the genitive here because reading genitives is harder mental work. it's a minor comment though and i don't mind if you keep it